client/src/components/video-thumbnail.jsx

```diff
@@ -29,9 +29,7 @@ const VideoThumbnail = ({ mentor, isMobile, width, height }) => {
   };
 
   const idleVideoId = findMentorIdleId(mentor);
-  const url = idleVideoId
-    ? videoUrl(mentor.id, idleVideoId, format)
-    : idleUrl(mentor.id, format);
+  const url = videoUrl(mentor.id, idleVideoId, format);
```

Comment:
Do we need this change? I think the old version was a safety fallback like 'if there's no video/id marked as `__IDLE__` in the metadata, just try a default url construct like `/mentors/clint/idle.mp4`